config/main.py

```diff
@@ -6515,7 +6515,9 @@ def add_ntp_server(ctx, ntp_ip_address):
         return
     else:
         try:
-            db.set_entry('NTP_SERVER', ntp_ip_address, {'NULL': 'NULL'})
+            db.set_entry('NTP_SERVER', ntp_ip_address,
+                         {'resolve_as': ntp_ip_address,
+                          'association_type': 'server'})
```

Comment:
Just to verify, this would mean that pool NTP servers can be configured only via `config_db.json` directly, right?

Reply:
Yes, right